tests/test_server.py

```diff
@@ -227,5 +249,14 @@ def test_get(self):
         self.assertEqual(metric, converted)
 
 
+def _create_corrupted_dataset(repo: MetricTestRepo) -> DatasetRef:
+    run = "corrupted-run"
+    ref = repo.addDataset({"instrument": "DummyCamComp", "visit": 423}, run=run)
+    uris = repo.butler.getURIs(ref, run=run)
```

Comment:
I'm hoping that the `run` parameter is not needed here because the `ref` knows its own run.